src/transformers/models/t5/modeling_t5.py

```diff
@@ -642,10 +668,12 @@ def forward(
         hidden_states, present_key_value_state = self_attention_outputs[:2]
         attention_outputs = self_attention_outputs[2:]  # Keep self-attention outputs and relative position weights
 
+        detect_overflow(hidden_states, "T5Block after T5LayerSelfAttention")
```

Comment:
Do we need to keep those statements in the code?

Reply:
none of those, as I mentioned in OP, I just kept them in case we need to change something during this PR so these immediately tell me if everything is still in norm.

So the plan is to delete them all once this PR is approved and before it's merged.